code/modules/photography/camera/other.dm

```diff
@@ -25,7 +25,13 @@
 	see_ghosts = CAMERA_SEE_GHOSTS_ORBIT
 
 /obj/item/camera/detective
-	name = "Detective's camera"
-	desc = "A polaroid camera with extra capacity for crime investigations."
+	name = "detective's camera"
+	desc = "A silent polaroid camera with extra capacity for crime investigations."
+	flash_enabled = FALSE
+	silent = TRUE
 	pictures_max = 30
 	pictures_left = 30
+
+/obj/item/camera/detective/after_picture(mob/user, datum/picture/picture)
+	. = ..()
+	user.playsound_local(src, pick('sound/items/polaroid1.ogg', 'sound/items/polaroid2.ogg'), 35, TRUE)
```

Comment:
You need to pass a turf to `playsound_local` or it won't sound as you would expect
```suggestion
	user.playsound_local(get_turf(src), pick('sound/items/polaroid1.ogg', 'sound/items/polaroid2.ogg'), 35, TRUE)
```